SPECS/ceph/CVE-2025-52939.patch

```diff
@@ -0,0 +1,39 @@
+From 5c7bec2d59bef1612f7a1f0a1acaa812572b92ee Mon Sep 17 00:00:00 2001
```

Comment:
Please use proper open-source community etiquette and restore the upstream community patch header information.  See https://github.com/lua/lua/commit/42d40581dd919fb134c07027ca1ce0844c670daf.patch